share/build-navigation/build-navigation-json.xsl

```diff
@@ -108,8 +114,152 @@
 
     <!-- Create individual JSON files for each docset -->
     <xsl:apply-templates select="//docset" mode="generate-docset-json"/>
+
+    <!-- Create individual site sections for each docset -->
+    <xsl:apply-templates select="/positivedocservconfig" mode="site-sections-json"/>
+
+  </xsl:template>
+
+
+  <xsl:template match="positivedocservconfig" mode="site-sections-json">
```

Comment:
tldr: I'd prefer not depending on this element name in any way, so we change it more easily later. `/*` should work, no?

Longread: The history of this element name is that once upon a time, Sven told me that he didn't want to handle the document-translation-list logic in Python, and I should just make sure to give him an easier-to-consume config. The translation markup used negative lists (i.e. a list of documents not translated into a certain language). Hence, after the transformation, we had positive lists (i.e. a list of translated documents) and that's where this name comes from.

The name is no longer accurate. After Eli complained that `negative-list` mode was extremely annoying, I added `positive-list` mode and finally removed `negative-list` altogether.

Reply:
> tldr: I'd prefer not depending on this element name in any way, so we change it more easily later. `/*` should work, no?

Good idea.
If we want to change this match from `positivedocservconf` to `*` you need to adapt the template rule in line 119 as well and change `/positivedocservconf`to `/*`.

It's implemented in commit 182dee6.